codecov/settings_base.py

```diff
@@ -526,6 +526,10 @@
             ),
         },
     )
+elif IS_DEV:
+    sentry_sdk.init(
```

Comment:
Do we also want the `dsn` param here?

https://spotlightjs.com/setup/other/

Reply:
I don't think so, in the docs it says:
> If you want your Sentry SDKs to talk to the Sidecar, enable the spotlight setting. You don’t need a dsn for Spotlight to work.
and we don't actually have a `SENTRY_DSN` in dev.